### PR TITLE
[DNM v2.6/v2.5.9] Add Rancher Webhook to default resourceset

### PR DESCRIPTION
**Pull request by nickgerace**
Dependent on #87 being merged, but is ready for review.

Note to reviewers: https://github.com/rancher/backup-restore-operator/pull/89#issuecomment-817950489

Relevant issues:
- #45
- #73

### CODE REVIEW

**Comment by nickgerace** [comment]
**Note to reviewers:** there is a service account, deployment, and service created in `cattle-system`. Since the `main.yaml` file covers all instances of those resources in that namespace, we do not need to add it to the new file.

File: charts/rancher-backup/files/default-resourceset-contents/main.yaml [hunk 1]
```diff
@@ -18,6 +18,9 @@
 - apiVersion: "v1"
   kindsRegexp: "^serviceaccounts$"
   namespaceRegexp: "^cattle-|^p-|^c-|^local$|^user-|^u-"
+- apiVersion: "v1"
+  kindsRegexp: "^services$"
+  namespaceRegexp: "^cattle-|^p-|^c-|^local$|^user-|^u-"
```
Comment (jiaqiluo) [inline charts/rancher-backup/files/default-resourceset-contents/main.yaml:23]: I think we should select only the webhook's one by calling out its name `rancher-webhook`. 
 ```suggestion
   namespaceRegexp: "^cattle-|^p-|^c-|^local$|^user-|^u-"
   resourceNames:
     - "rancher-webhook"
```

**Review by jiaqiluo — CHANGES_REQUESTED** [review]

File: charts/rancher-backup/files/default-resourceset-contents/main.yaml (hunk 1 above)
Comment (nickgerace) [reply charts/rancher-backup/files/default-resourceset-contents/main.yaml:23]: Hmm, probably a good idea. However, we should keep deployments and serviceaccounts the same.

Comment (jiaqiluo) [reply charts/rancher-backup/files/default-resourceset-contents/main.yaml:23]: it is a good point.  
It is ok that we just backup all services under the target namespaces if that is how we select other resources

**Review by jiaqiluo — APPROVED** [review]
LGTM

**Review by ryansann — APPROVED** [review]

**Comment by nickgerace** [comment]
Closing since https://github.com/rancher/backup-restore-operator/issues/73 has been closed

**Pull request closed without merging**